lesson02-homework/src/main/java/com/haubui/view/Ex07.java

```diff
@@ -0,0 +1,30 @@
+package com.haubui.view;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class Ex07 {
+	public static void main(String[] args) {
+		Random rd = new Random();
+		int[] numbers = new int[5];
+		String temp = "";
```

Comment:
It's your good idea. To check whether does element exist in the numbers array ?
But you have to create so many  memory for the 'String'. Because It's immutable and always create new memory when it has the new value.
2 ways
1. Using StringBuilder (single - thread)
2. Remove it and using array online. Check the elements exist in the array as normal

Reply:
oh, i don't think of memory
I fixed it with StringBuilder